machineconfiguration/v1alpha1/techpreview.machineconfignode.testsuite.yaml

```
@@ -0,0 +1,85 @@
+apiVersion: apiextensions.k8s.io/v1 # Hack because controller-gen complains if we don't have this
+name: "[Custom] MachineConfigNode"
+crd: 0000_80_machineconfignode-CustomNoUpgrade.crd.yaml
```

Should be the techpreview CRD
```suggestion
crd: 0000_80_machineconfignode-TechPreviewNoUpgrade.crd.yaml
```